spaces/domains.json

```
@@ -3,6 +3,7 @@
   "snapshot.gasdao.org": "gasdao.eth",
   "vote.bomb.money": "1coinnetwork.eth",
   "vote.chainscore.finance": "chainscore.eth",
+  "theartistdao.com": "theartistdao.eth",
```

You should use a subdomain.
```suggestion
  "vote.theartistdao.com": "theartistdao.eth",
```

---

Is this a requirement? We are not using this domain for anything other than this purpose at this time.

---

This is not a requirement, it would work well with the root domain theartistdao.com